LICENSE

```diff
@@ -1,7 +1,15 @@
+BSD 3-Clause License
+
+From napari-animation:
 
 Copyright (c) 2020, Nicholas Sofroniew
 All rights reserved.
 
+From naparimovie:
```

Comment:
cc @guiwitz I added this copy right notice to reflect the input from naparimovie